plugin/adsk/scripts/AEmayaUsdProxyShapeBaseTemplate.mel

```diff
@@ -26,6 +26,36 @@ global proc AEmayaUsdProxyShapeInfoReplace(string $input)
     textFieldGrp -edit -text $defaultPrim mayaUsdProxyShapeDefaultPrim;
 }
 
+global proc AEmayaUsdProxyShapePayloadsNew(string $loadPayloads)
+{
+    setUITemplate -pst attributeEditorTemplate;
+    rowLayout -nc 5 -cl5 "right" "center" "center" "center" "center";
+        text -label `getMayaUsdString("kPayloads")` -ann `getMayaUsdString("kPayloadsAnn")`;
+        button -label `getMayaUsdString("kLoadAll")` payloadsLoadAllBtn;
+        button -label `getMayaUsdString("kUnloadAll")` payloadsUnloadAllBtn;
+        text -label "";
+        text -label "";
```

Comment:
Use the same trick as Maya source does to get things to line up. I have a text field with two buttons (on a row with 5 columns).